### DISCUSSION TITLE
github copolit

### DISCUSSION THREAD

**Discussion by 21bb** [community/community#129409] [GitHub Education] [answered]
Hello, I'm applying for github in copolit, there has never been an interface for me to upload Xuexin online student status certification. Instead, there were questions about why I wasn't at school, I'm at school now, and I really want to apply for copolit to help me study. Hoping to open up upload permissions for me, so I can apply for copolit, thank you.

**Comment by Shubham-Saxena-16** [comment]
Sure, a formal message you can use to request upload permissions for your Xuexin online student status certification

**Comment by 21bb** [comment]
Hi,How to use the official message to request the permission to upload the
online student identity certificate of Xuexin

Shubham Saxena ***@***.***> 于 2024年6月22日周六 15:55写道：

**Comment by Shubham-Saxena-16** [comment, marked as answer]
Great if I was able to answer your question please mark your question as answered

**Reply by 21bb** [reply]
Hello, I have marked as answered, I log into github education, there is still no page for me to upload Xuexin Network Student Status Report. Please help me solve it. Thank you

**Reply by 21bb** [reply]
Hello, I have marked as answered, I log into github education, there is still no page for me to upload Xuexin Network Student Status Report. Please help me solve it. Thank you
